src/test/java/ttt/BoardSizeTest.java

```diff
@@ -21,17 +20,17 @@
         });
     }
 
-    private int fInput;
-    private int fExpected;
+    private int size;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/mollie-c-stephenson/TicTacToe/file/2695155926/issues/source?bid=3284885&fileBranchId=3339873#l23)